### PR TITLE
Oleksandr Kovaliuk 1.1: matchResult method in FootballMatch class

### PR DESCRIPTION
**Pull request by Oleksandr58**
## 💡️ Motivation and Context

Motivation: 1.1
Context: `matchResult method in FootballMatch class`.

## 🧪 How was this Pull Request tested?

* matchResult method
   * [x] Manual
   * [x] Unit
   * [ ] Integration

### CODE REVIEW

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 1]
```diff
@@ -15,6 +15,10 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        return result1 == bet1 && result2 == bet2 ? 2 : 
+                    ((result1 > result2 && bet1 > bet2)
+                        || (result1 < result2 && bet1 < bet2) 
+                        || (result1 == result2 && bet1 == bet2)
+                    ? 1 : 0);
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FootballMatch.java]:
```suggestion
                        || (result1 == result2 && bet1 == bet2) ? 1 : 0);
```
try not to split lines too much

Comment (Oleksandr58) [reply src/main/java/core/basesyntax/FootballMatch.java]: Thanks, done

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 2]
```diff
@@ -15,6 +15,9 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        return result1 == bet1 && result2 == bet2 ? 2 : 
+                    ((result1 > result2 && bet1 > bet2)
+                        || (result1 < result2 && bet1 < bet2) 
+                        || (result1 == result2 && bet1 == bet2) ? 1 : 0);
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FootballMatch.java]:
```suggestion
                    (result1 > result2 && bet1 > bet2) || 
                    (result1 < result2 && bet1 < bet2) || 
                    (result1 == result2 && bet1 == bet2) ? 1 : 0;
```
what are brackets for?

**Review by Tarabarnik — APPROVED** [review]

**Comment by Tarabarnik** [comment]
your code did not pass tests. I can not approve this

**Review by Tarabarnik — APPROVED** [review]

**Pull request closed without merging**